parcelgen.py

```diff
@@ -77,7 +77,7 @@ def array_type(self, typ):
     def map_type(self, typ):
         match = re.match(r"(Map)<(.*, .*)>", typ)
         if match:
-            split = match.group(2).split(', ')
+            split = match.group(2).split(', ', 1)
```

Comment:
I'd consider adding a comment here about why we have this 1

Reply:
@maltzj 
The input string is something like "String, Map<String, Something>"
it would split it to [String", "Map<String", "Something>"]
the first element taken as the key type, the second element is taken as the value type, so this results in a value type of Map<String, which causes a syntax error
The correct split is to only split at the first comma:
["String", "Map<String, Something>"]
and we get the correct value type